build.gradle.kts

```diff
@@ -9,5 +9,5 @@ plugins {
 }
 
 allprojects {
-    version = "0.2.2"
+    version = "0.2.3"
```

Comment:
This is now automated with a manual CI release task! So we don't want to manually bump the version in PRs anymore.

Reply:
oh, oops!